### DISCUSSION TITLE
GoPro HERO9 as a dashcam

### DISCUSSION THREAD

**Discussion by maxserg** [gopro/labs#897] [Q&A] [answered]
Hi all, I'm looking for some advice and help on using GoPro HERO9 as a dashcam. I found this script https://www.youtube.com/watch?v=T87RekjaVA4 very interesting and useful, but unfortunately, it's designed for Hero11/12 series.
Actually, I just need to have:
1) GoPro turns on when power is on
2) GoPro starts recording while moving (acceleration or g-sensor trigger?)
3) GoPro stops recording and shutdowns itself after 1 minute when power is off (or immediately, it's not necessary to wait 1 minute)

I do believe Looping profile needs to be preconfigured preliminary. Can someone please direct me how to build a required QR code for this quite simple scenario?

**Comment by dnewman-gpsw** [comment, marked as answer]
It can't do all those things on HERO9.  USB Power auto record is all it can do : https://gopro.github.io/labs/control/usb/  
To combine IMU triggers with power trigger need to the newer Labs scripting features of HERO11 and 12.

**Reply by maxserg** [reply]
So, sounds like the only single action https://[gopro.github.io/labs/control/usb/](https://gopro.github.io/labs/control/usb/) can do all of the 3 required steps? Looks cool either :)

**Reply by dnewman-gpsw** [reply]
It is little basic, but works.  No IMU trigger.

**Reply by TripleFJeff** [reply]
I'm doing this with a HERO8-- power on with USB and start on GPS speed. The only exception is that it doesn't power off when it loses USB power. It just stays on until the battery dies. I suppose if you took the battery out, it would do what you want. Look for some of my posts here.
